src/data/network/pci_subclasses.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
We read it from hwinfo and subclasses so for network PCI class 0x02 ("network") those are the recognized ones.

https://pci-ids.ucw.cz/read/PD